modules/AlphaWalletFoundation/AlphaWalletFoundation/KeyManagement/EtherKeystore.swift

```diff
@@ -169,7 +169,7 @@ open class EtherKeystore: NSObject, Keystore {
         DispatchQueue.global(qos: .userInitiated).async { [weak self] in
             guard let strongSelf = self else { return }
 
-            let mnemonicString = strongSelf.generateMnemonic()
+            let mnemonicString = strongSelf.generateMnemonic(seedPhraseCount: .word12, passphrase: strongSelf.emptyPassphrase)
```

Comment:
Maybe `createAccount(completion:)` can reuse some of the logic starting here https://github.com/AlphaWallet/alpha-wallet-ios/pull/6019/files#diff-a24e213e54bb6382ad3d580d52df835f26ce0c8f996220c0d9ec2aa65e86c877R254 ?

Reply:
not clearly understood, could u please describe more

Reply:
The createAccount() func can be replaced too? Since some of the code is duplicated.

Reply:
Ah, ok. Never mind then. Can look at it later. Maybe add a TODO to this line number: https://github.com/AlphaWallet/alpha-wallet-ios/pull/6019/files#diff-a24e213e54bb6382ad3d580d52df835f26ce0c8f996220c0d9ec2aa65e86c877R255